src/terminal_renderer/Renderer.cpp

```diff
@@ -107,7 +117,7 @@ unique_ptr<text::shaper> createTextShaper(TextShapingEngine _engine, crispy::Poi
             #if defined(_WIN32)
             debuglog(TextRendererTag).write("Using DirectWrite text shaping engine.");
             // TODO: do we want to use custom font locator here?
-            return make_unique<text::directwrite_shaper>(_dpi);
+            return make_unique<text::directwrite_shaper>(_dpi, std::move(_locator));
```

Comment:
```suggestion
            return make_unique<text::directwrite_shaper>(_dpi, move(_locator));
```

As `_locator` is of type `std::unique_ptr<>` I think ADL should take effect and `std::` is not needed. :)